dgreen15/updating-onboarding-description (#31168)
DataDog/documentation: content/en/internal_developer_portal/software_catalog/entity_model/entity_types.md

    - service:payment-service
  componentOf:
    - system:shopist-checkout-platform
```

{{% /collapse-content %}}

{{% collapse-content title="YAML for manually defined frontend app" level="h4" expanded=false id="manually-created" %}}

This example shows a `kind:frontend` definition for a manually declared frontend app.

### Example YAML definitions
```yaml
apiVersion: v3
kind: frontend
metadata:
  name: checkout-webapp
  displayName: Checkout Web App
  description: Main frontend experience for the checkout flow in Shopist
  owner: shopist-frontend
  additionalOwners:
    - name: ux-platform-team
  type: team
  links:
    - name: "UX Design Guidelines"
      type: doc
      url: https://wiki.internal/checkout-design
    - name: "Frontend Source Code"
      type: repo
      provider: github
      url: https://github.com/shopist/checkout-webapp
spec:
  type: browser
  lifecycle: production
  tier: tier1
  dependsOn:
    - service:checkout-api
    - service:payment-service
  componentOf:
    - system:shopist-checkout-platform
```

{{% /collapse-content %}}

When this definition is created:

- The frontend app appears under the Frontend Apps section in Software Catalog.

> - If a RUM application exists with the same name or ID, its telemetry is automatically linked. You can find the name under [Manage Applications][1] or the ID will be autofilled when you click **Add Metadata** on an existing Frontend App in Software Catalog.

Reviewer: ```suggestion
- If a RUM application exists with the same name or ID, its telemetry is automatically linked. You can find the name and ID under [Manage Applications][1], or you can click **Add Metadata** on an existing frontend app in Software Catalog to autofill the ID.
```
